File: Supermercado/Supermercado2.cs
//Supermercado con Logica C Sharp
using System;

namespace ConsoleApp1
{
    class Program
    {

        static void Main(string[] args)
        {

            while (true)
            {
                string entrada = Console.ReadLine()!;
                if (string.IsNullOrEmpty(entrada))
                {
                    break;
                }
                else
                {
                    char es = ' ';
                    char tab = '\t';
                    int sum = 0;
                    string local = string.Empty;

                    for (int j = 0; j < entrada.Length; j++)
                    {
                        if (entrada[j] != es && entrada[j] != tab)
                        {
                            local += entrada[j];
                            if (j == entrada.Length - 1)
                            {
                                sum += int.Parse(local);
                            }
                        }
                        else
                        {
                            if (local != "")
                            {
                                sum += int.Parse(local);
                                local = string.Empty;
                            }
                        }
                    }
                    Console.WriteLine(sum);
                }
            }
        }
    }
}

File: Supermercado/Supermercado2.py
#supermercado python con logica
import sys
#import re

conta = int(0)
while True:
    try:
      cad = input()
    except EOFError:
      break
    
    if not cad:
        # la cadena vacia...
        sys.exit()
    else:
        es = ' '
        tab = '\t'
        conta = 0
        local = str()
        nocad = str()
        
        for i in range(0, len(cad), 1):
            if cad[i] != es and cad[i] != tab:
                local = local + cad[i]
                if i == len(cad)-1:
                    conta = conta + int(local)
            else:
                if local != nocad:
                    conta = conta + int(local)
                    local = str()
        print(conta)
#end program